src/wazuh.py

```diff
@@ -35,22 +36,47 @@ class WazuhInstallationError(Exception):
     """Base exception for Wazuh errors."""
 
 
-def update_configuration(container: ops.Container, indexer_ips: list[str]) -> None:
-    """Update Wazuh configuration.
+class NodeType(Enum):
+    """Enum for the W<çazuh node types.
+
+    Attrs:
+        WORKER: worker.
+        MASTER: master.
+    """
+
+    WORKER = "worker"
+    MASTER = "master"
+
+
+def _update_filebeat_configuration(container: ops.Container, ip_ports: list[str]) -> None:
+    """Update Filebeat configuration.
 
     Arguments:
         container: the container for which to update the configuration.
-        indexer_ips: list of indexer IPs to configure.
-
-    Raises:
-        WazuhInstallationError: if an error occurs while installing.
+        ip_ports: list of indexer IPs and ports to configure.
     """
-    ip_ports = [f"{ip}" for ip in indexer_ips]
     filebeat_config = container.pull(FILEBEAT_CONF_PATH, encoding="utf-8").read()
     filebeat_config_yaml = yaml.safe_load(filebeat_config)
     filebeat_config_yaml["output.elasticsearch"]["hosts"] = ip_ports
     container.push(FILEBEAT_CONF_PATH, yaml.safe_dump(filebeat_config_yaml), encoding="utf-8")
 
+
+def _update_wazuh_configuration(  # pylint: disable=too-many-locals
```

Comment:
Can you add a comment to justify why you need to disable `too-many-locals` ?
Or refactor the method to avoid disabling it